cmd/porter/bundle.go

```diff
@@ -108,7 +108,7 @@ For example, the 'debug' driver may be specified, which simply logs the info giv
   porter bundle install MyAppFromTag --tag getporter/kubernetes:v0.1.0
 `,
 		PreRunE: func(cmd *cobra.Command, args []string) error {
-			return opts.Validate(args, p.Context)
+			return opts.Validate(args, p.CNAB, p.Context)
```

Comment:
If you want to simplify, in other places I have passed the Porter app `p` to `Validate` in cases where I need to pass multiple components that porter controls. You don't have to, just letting you know that we've done that before.